Dockerfile

```
@@ -1,5 +1,11 @@
 # stage 1 Generate celestia-appd Binary
 FROM --platform=$BUILDPLATFORM docker.io/golang:1.21.3-alpine3.18 as builder
+
+ARG TARGETOS
+ARG TARGETARCH
```

[optional] it looks like ARGs can contain underscores, in which case this seems a bit easier to read but def not blocking

Ref:

https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
```suggestion
ARG TARGET_OS
ARG TARGET_ARCH
```